meinberlin/apps/plans/views.py

```diff
@@ -47,6 +47,36 @@ def get_districts(self):
         except ObjectDoesNotExist:
             return []
 
+    def _get_status_string(self, projects):
```

Comment:
like that:
`if project:
    find status_string and status_class
else:
    if item.participation:
        status_string = planned
     else:
        something else`